extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -222,6 +237,37 @@ const stateComparisonFunctions = {
     (old, curr) => old.element !== curr.element || old.state !== curr.state,
   [StateProperty.NAVIGATION_PATH]: (old, curr) => old.length !== curr.length,
   [StateProperty.PAGE_IDS]: (old, curr) => old.length !== curr.length,
+  [StateProperty.INTERACTIVE_RESULTS_STATE]: (old, curr) => {
+    return old.finished != curr.finished;
```

Comment:
Don't you need to know if the `category` got updated as well, to re-render the results component even if all the polls weren't answered?

Reply:
That's right, this implementation used to only update the results component when you finished all the results, but now we want to show the results even if you haven't answered all the polls. I'll work on it